vendor/assets/javascripts/brainstem/loaders/abstract-loader.coffee

```diff
@@ -318,9 +318,12 @@ class Brainstem.AbstractLoader
   _modelsOrObj: (obj) ->
     if obj instanceof Backbone.Collection
       obj.models
+    else if obj instanceof Array
+      obj
+    else if obj
+      [obj]
     else
-      obj || [] # TODO: revisit this.. we shouldn't be getting to this stage.
```

Comment:
I wonder why it was thought that reaching this fork in the road shouldn't be possible? Perhaps our problem is a bit more fundamental to brainstem-js / mavenlink-js ?